hack/tests/e2e-ansible.sh

```diff
@@ -175,28 +166,36 @@ test_operator() {
 
 header_text "Creating and building the operator"
 pushd "$TMPDIR"
-operator-sdk new memcached-operator \
-  --api-version=ansible.example.com/v1alpha1 \
-  --kind=Memcached \
-  --type=ansible
-cp "$ROOTDIR/test/ansible-memcached/tasks.yml" memcached-operator/roles/memcached/tasks/main.yml
-cp "$ROOTDIR/test/ansible-memcached/defaults.yml" memcached-operator/roles/memcached/defaults/main.yml
-cp -a "$ROOTDIR/test/ansible-memcached/memfin" memcached-operator/roles/
-cat "$ROOTDIR/test/ansible-memcached/watches-finalizer.yaml" >> memcached-operator/watches.yaml
-# Append Foo kind to watches to test watching multiple Kinds
-cat "$ROOTDIR/test/ansible-memcached/watches-foo-kind.yaml" >> memcached-operator/watches.yaml
-
-
+mkdir memcached-operator
 pushd memcached-operator
-
+operator-sdk init --plugins ansible.operator-sdk.io/v1 \
+  --domain example.com \
+  --group ansible \
+  --version v1alpha1 \
+  --kind Memcached \
+  --generate-playbook \
+  --generate-role
+cp "$ROOTDIR/test/ansible-memcached/tasks.yml" roles/memcached/tasks/main.yml
+cp "$ROOTDIR/test/ansible-memcached/defaults.yml" roles/memcached/defaults/main.yml
+cp -a "$ROOTDIR/test/ansible-memcached/memfin" roles/
+marker=$(tail -n1 watches.yaml)
+sed -i'.bak' -e '$ d' watches.yaml;rm -f watches.yaml.bak
+cat "$ROOTDIR/test/ansible-memcached/watches-finalizer.yaml" >> watches.yaml
+echo $marker >> watches.yaml
 header_text "Adding a second Kind to test watching multiple GVKs"
-operator-sdk add crd --kind=Foo --api-version=ansible.example.com/v1alpha1
-sed -i".bak" -E -e 's/(FROM quay.io\/operator-framework\/ansible-operator)(:.*)?/\1:dev/g' build/Dockerfile; rm -f build/Dockerfile.bak
-operator-sdk build "$DEST_IMAGE"
+operator-sdk create api --kind=Foo --group ansible --version=v1alpha1
+sed -i".bak" -e 's/# FIXME.*/role: \/dev\/null/g' watches.yaml;rm -f watches.yaml.bak
+
+sed -i".bak" -E -e 's/(FROM quay.io\/operator-framework\/ansible-operator)(:.*)?/\1:dev/g' Dockerfile; rm -f Dockerfile.bak
+IMG=$DEST_IMAGE make docker-build
 # If using a kind cluster, load the image into all nodes.
 load_image_if_kind "$DEST_IMAGE"
-sed -i".bak" -E -e "s|REPLACE_IMAGE|$DEST_IMAGE|g" deploy/operator.yaml; rm -f deploy/operator.yaml.bak
-sed -i".bak" -E -e 's|Always|Never|g' deploy/operator.yaml; rm -f deploy/operator.yaml.bak
+pushd config/testing
+kustomize edit set image testing=$DEST_IMAGE
+kustomize edit add patch pull_policy/Never.yaml
+kustomize edit set namespace default
+popd
+
```

Comment:
We should use the default makefile targets here that should work equals accross the projects. See: https://github.com/operator-framework/operator-sdk/blob/c1b615c6f01d303ece5bcb06f46565ae0b6b7999/hack/tests/e2e-helm.sh#L143-L157